sonar-python-plugin/src/test/java/org/sonar/plugins/python/indexer/SonarLintPythonIndexerProjectLevelSymbolTableBuildingTest.java

```diff
@@ -0,0 +1,121 @@
+/*
```

Comment:
Note: I have the feeling that, as we don't use keep dedicated V2 and V1 APIs for descriptors, this test is redundant, since existing tests on indexing will cover its use cases.
I'd like your opinion before removing it though @maksim-grebeniuk-sonarsource & @thomas-serre-sonarsource.

Reply:
Yep I m fine to remove those tests! :+1: